src/custom/components/SearchModal/CommonBases/index.ts

```diff
@@ -30,7 +30,7 @@ export const AutoColumn = styled(AutoColumnUni)`
 
 export const MobileWrapper = styled(AutoColumn)`
   ${({ theme }) => theme.mediaWidth.upToSmall`
-    isplay: none;
```

Comment:
Actually in mobile devices we can see the common tokens row. 
![image](https://user-images.githubusercontent.com/622217/154729569-80161d8f-bb1f-40ed-a9b1-c4fd8f7c5856.png)

What should we do with this? 
If we fix this typo with `display: none;` , the common token list will disappear:
![image](https://user-images.githubusercontent.com/622217/154729118-02088650-5955-43d5-bcca-de045f9bda7d.png)

I believe we can keep it.
cc @elena-zh @alfetopito @michelbio